modules/core/LocationManager.js

```diff
@@ -263,7 +263,7 @@ export class LocationManager extends EventEmitter {
    * Returns a locationSetID for a given locationSet (fallback to `+[Q2]`, world)
    * (The locationSet doesn't necessarily need to be resolved to compute its `id`)
    *
-   * @param  `locationSet`  A locationSet Object, e.g. `{ include: ['us'] }`
+   * @param  `locationSet`  A locationSet {Set} Object, e.g. include: [ 'us' ]
```

Comment:
This change is not correct

Reply:
This was one of the 'formatting' issues that jsdoc kept emitting at me- it thinks that the thing in curly braces is a type definition {}. I'll see if I can figure out how to just annotate this as a literal string that it should ignore.

Reply:
Did this one better- I created a typeDef for the locationSet object in comments, and now jsdoc seems happy.